tests/ui/redundant_field_names.stderr

```diff
@@ -0,0 +1,16 @@
+error: redundant field names in struct initialization
+  --> $DIR/redundant_field_names.rs:22:17
+   |
+22 |         gender: gender,
+   |                 ^^^^^^ help: replace 'gender: gender' with 'gender'
```

Comment:
This will most likely not yield an auto-fixable suggestion. You'll need to remove the colon + space as well for that.

Reply:
Is this ok ?

```
error: redundant field names in struct initialization
  --> tests/ui/redundant_field_names.rs:23:9
   |
23 |         age: age,
   |         ^^^^^^^^ help: replace it with with 'age'
```

Reply:
Looks good!